css/style.css

```diff
@@ -79,6 +106,77 @@ footer {
 /* ADDITIONAL STYLES
 /*******************************************/
 
+.slogan {
+  font-size: 24px;
+  text-align: center;
+  font-weight: 600;
+  color: white;
+  text-align: center;
+  padding-top: 60px;
+}
+
+.button1 {
+  background-color: #f9e42e; /*yellow*/
+  border: none;
+  color: #033048; /*dark blue*/
+  padding: 15px 50px;
+  text-decoration: none;
+  font-size: 16px;
+  font-weight: 600px;
+}
+
+.benefits {
+    background-color: #f0efef;
```

Comment:
Try to make sure your indentation is consistent in both your HTML and CSS files (and JavaScript)